app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -469,6 +469,8 @@ class LuciBuildService {
       'commit/gitiles/chromium.googlesource.com/external/github.com/flutter/$repo/+/$commitSha',
     ];
     tags['user_agent'] = <String>['luci-scheduler'];
+    // Tag `scheduler_job_id` is needed when calling buildbucket search build API.
+    tags['scheduler_job_id'] = <String>[builderName];
```

Comment:
I believe we also need the repository. E.g 'flutter/Linux Android AOT Engine'

Reply:
Good catch! Added.